addons/account_edi_ubl_generic/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Import/Export invoices with generic UBL',
+    'description': '''
+    Support for Export/Import in UBL format (2.1) and BIS3 (EN16931) format.
+    ''',
+    'version': '1.0',
+    'category': 'Accounting/Accounting',
+    'depends': ['account_edi_ubl'],
```

Comment:
this is a bit confusing to me: Shouldn't `account_edi_ubl_generic` be the generic part and `account_edi_ubl` be the override creating the `account.edi.format`?

Reply:
Why not:
`account_edi_ubl_base`: common templates used by everything using ubl
`account_edi_ubl`: UBL edi.format
`account_edi_ubl_bis3`: extension of `account_edi_ubl_base` to be bis3 compliant?